cli/config/configfile/file.go

```diff
@@ -43,6 +43,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	BuildStreamContext   *bool                       `json:"buildStreamContext,omitempty"`
```

Comment:
I guess most of these start with the object name (service, images, etc). but `buildStreamContext` doesn't read too well.

How about `buildAlwaysStreamContext` ? That way it's a little more clear what this is enabling.